src/cljs/rems/administration/create_form.cljs

```diff
@@ -660,22 +664,23 @@
            [:div.form-field {:id (field-editor-id (:field/id field))
                              :key index
                              :data-field-index index}
-            [:div.form-field-header
-             [:h3 (text-format :t.create-form/field-n (inc index) (localized-field-title field @(rf/subscribe [:language])))]
-             [:div.form-field-controls
+            [:div.form-field-header.d-flex
+             [:h3
+              (text-format :t.create-form/field-n (inc index) (localized-field-title field @(rf/subscribe [:language])))]
+             [:div.form-field-controls.text-nowrap.ml-auto
```

Comment:
In the original I prefer the `h3` oneliner. For me it reads that ok here's the `h3` and I don't need to care about the rest. With the contents split to the next row I need to visually check that the one character indentation is there.